pages/app/models/refinery/page.rb

```diff
@@ -376,9 +364,9 @@ def puts_destroy_help
     end
 
     def slug_locale
-      return Globalize.locale if translation_for(Globalize.locale, false).try(:slug).present?
+      return Mobility.locale if slug_backend.read(Mobility.locale).present?
```

Comment:
I'm not quite sure about the context, but `slug_backend.read(Mobility.locale)` is basically the same as `slug` here. This will build the translation for that locale if none exists yet.

With Globalize you have `translation_for(Globalize.locale, false)` which will avoid building the translation for the current locale if none yet exists. This is a bit different.

Mobility does expose a method `translation_for` [on the backend](https://github.com/shioyama/mobility/blob/b42686f65fa3eb98ed1c9bded6ded8a1c816f2d0/lib/mobility/backends/active_record/table.rb#L162-L166), but it has no option to override building an empty translation. You shouldn't really need that option though because [Mobility anyway removes any empty translations before saving](https://github.com/shioyama/mobility/blob/b42686f65fa3eb98ed1c9bded6ded8a1c816f2d0/lib/mobility/backends/active_record/table.rb#L168-L177), so I think here you should be able to safely replace `slug_backend.read(Mobility.locale)` with just `slug`, and `slug_backend.read(Refinery::I18n.default_frontend_locale)` with just `slug(locale: Refinery::I18n.default_frontend_locale)`.

You don't really need `present?` either since the [Presence plugin](https://github.com/shioyama/mobility/blob/b42686f65fa3eb98ed1c9bded6ded8a1c816f2d0/lib/mobility/plugins/presence.rb) converts any blank strings to `nil` for you already.